av1an-cli/src/lib.rs

```diff
@@ -609,6 +609,14 @@ pub fn parse_cli(args: CliOpts) -> anyhow::Result<EncodeArgs> {
   Ok(encode_args)
 }
 
+///  Make make sure splits are not longer than 10 seconds
+pub fn calculate_extra_splits_from_fps(fps: anyhow::Result<f64>) -> usize {
```

Comment:
This function probably doesn't need to exist, can we move this code into where the function is being called?